ci_framework/roles/hci_prepare/README.md

```diff
@@ -0,0 +1,42 @@
+# hci_prepare
+Prepare steps and files needed when deploying Ceph services alongside with compute nodes using `edpm_deploy` role.
+
+## Privilege escalation
+None.
+
+## Parameters
+* `cifmw_hci_prepare_basedir`: (String) Base directory. Defaults to `cifmw_basedir` which defaults to `~/ci-framework-data`.
+* `cifmw_hci_prepare_dataplane_dir`: (String) Directory in where `edpm_deploy` role will search for DataPlane kustomizations. Defaults to `"{{ cifmw_basedir }}/artifacts/manifests/kustomizations/dataplane"`.
+* `cifmw_hci_prepare_dryrun`: (Boolean) Perform a dry run on a set of commands. Defaults to `False`.
+* `cifmw_hci_prepare_skip_load_parameters`: Skip the initial `load_parameters` step, which load vars to gather network information. Defaults to `False`.
+* `cifmw_hci_prepare_ceph_secret_path`: "/tmp/k8s_ceph_secret.yml"
+* `cifmw_hci_prepare_enable_storage_mgmt`: (Boolean) Creates a kustomization file to include `storage-mgmt` network in DataPlane deployment. Defaults to `True`.
+* `cifmw_hci_prepare_storage_mgmt_mtu`: (Int) Storage-Management network MTU. Defaults to `1500`.
+* `cifmw_hci_prepare_storage_mgmt_vlan`: (Int) Storage-Management network VLAn. Defaults to `23`.
+
+## Examples
```

Comment:
Thank you for this nice overview of how it works.

Reply:
indeed! I guess we can copy/paste in follow ups to add the nfs network for ganesha/manila?

cc @gouthampacha

Reply:
Yes, nfs/ganesha could be a follow up. The next step which should work quickly though is to add native CephFS manila.

When this patch gets a follow up so that the contorl plane also uses Ceph, manila with CephFS can be added by using this kustomize.

https://github.com/fultonj/antelope/blob/main/crs/control_plane/overlay/ceph/deployment.yaml#L33-L74